ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -10,29 +14,37 @@
 
 public class DriveCode {
 
-    private static final FloatInput driveLeftAxis = ZukoAzula.controlBinding.addFloat("Drive Left Axis").deadzone(0.2f);
-    private static final FloatInput driveRightAxis = ZukoAzula.controlBinding.addFloat("Drive Right Axis").deadzone(0.2f);
+    private static final FloatInput driveLeftAxis = ZukoAzula.controlBinding.addFloat("Drive Left Axis").deadzone(0.2f).negated();
+    private static final FloatInput driveRightAxis = ZukoAzula.controlBinding.addFloat("Drive Right Axis").deadzone(0.2f).negated();
     private static final FloatInput driveRightTrigger = ZukoAzula.controlBinding.addFloat("Drive Forwards Trigger").deadzone(0.2f);
     private static final FloatInput driveLeftTrigger = ZukoAzula.controlBinding.addFloat("Drive Backwards Trigger").deadzone(0.2f);
 
     private static final ExtendedMotor rightFrontCAN = FRC.talonCAN(2), rightBackCAN = FRC.talonCAN(3);
     private static final ExtendedMotor leftFrontCAN = FRC.talonCAN(0), leftBackCAN = FRC.talonCAN(1);
 
-    public static void setup() throws ExtendedMotorFailureException {
+    private static final BehaviorArbitrator behaviors = new BehaviorArbitrator("Drive Code");
+    private static final ArbitratedFloat leftInput = behaviors.addFloat();
+    private static final ArbitratedFloat rightInput = behaviors.addFloat();
+    private static final Behavior teleop = behaviors.addBehavior("Teleop", FRC.inTeleopMode());
+    private static final Behavior pit = behaviors.addBehavior("Pit Mode", ZukoAzula.mainTuning.getBoolean("Pit Mode Enable", false));
```

Comment:
Why is pit mode a tunable value? Also, what happens if we start a match with pit mode enabled?